Bill3.java

```diff
@@ -1,8 +1,14 @@
 // Splits a restaurant bill evenly among three diners.
 public class Bill3 {
 	public static void main(String[] args) {
-		// To get you started, here is the first line in the program:
-	    String name1 = args[0];
-	    // Replace this comment with the rest of your code   
+		// defines 3 names
+	    String n1 = args[0];
+		String n2 = args[1];
```

Comment:
n1, n2 and n3 are not understandable variable names.
You could have named them: "name1", "name2", "name3", or for example: "firstName", "secondName", "thirdName".
What matters is that other people who reads your code would not have to make an effort to understand what this variables represents.